S4TF_Tutorial_1.ipynb

```diff
@@ -331,7 +331,7 @@
         "\n",
         "Python has lists built into the language.\n",
         "However, we will use a library called numpy for this.\n",
-        "Numpy gives you lot's of support functions that are useful when doing Machine Learning.\n",
+        "Numpy gives you lots of support functions that are useful when doing Machine Learning.\n",
```

Comment:
In this, and throughout the notebook, it might be best to capitalize NumPy per the official capitalization of the project: https://www.numpy.org/